commands/build.py

```diff
@@ -93,13 +139,57 @@ def __init__(self, args):
                 builder.prepare()
                 builder.make()
 
+    @staticmethod
+    def build_deps(args):
+        if args.verbose: logger.setLevel(logging.DEBUG)
+        commands = args.commands
+        modules = deps_modules if 'all' in args.modules else [m for m in deps_modules if m in args.modules]
+        modules_dir = os.path.join(os.path.realpath(args.map_path), "../3rdparty")
+        build_dir = os.path.realpath(modules_dir + '/../multiap/build')
+        install_dir = os.path.realpath(build_dir + '/install')
+
+        if 'distclean' in commands and os.path.exists(build_dir):
+             logger.info("distclean - deleting {}".format(build_dir))
```

Comment:
This will delete everything, including the map build dir. Also, this is duplicating code which exists in build_map. See the previous comment regarding unifying this using inheritance and polymorphism